src/components/App.js

```diff
@@ -1,28 +1,164 @@
-import React, { useState, useEffect } from 'react';
+import React, { Fragment, useState, useEffect } from 'react';
+import { NavLink, Route, useHistory } from 'react-router-dom';
 
 import {
-  getSomething
+  getAllProducts,
+  getAllOrders
 } from '../api';
 
-const App = () => {
-  const [message, setMessage] = useState('');
+import { 
+  Product, 
+  AllProducts, 
+  Nav, 
+  Login, 
+  Register, 
+  AllOrders, 
+  SingleOrder, 
+  Cart,
+  Footer, 
+  SingleProduct,
+  SingleUser } from './index';
+
+// import SingleOrder from './SingleOrder';
+import './App.css';
+
+const App = (props) => {
+  const {orderId, userId, cart, setCart, singleProduct, setSingleProduct, singleUser, setSingleUser} = props;
+  const [user, setUser] = useState('');
+  const [token, setToken] = useState('');
+  const [nav, setNav] = useState([]);
+  const [product, setProduct] = useState([]);
+  const [allProducts, setAllProducts] = useState([]);
+  const [singleOrder, setSingleOrder] = useState([]);
+  const [allOrders, setAllOrders] = useState([]);
 
   useEffect(() => {
-    getSomething()
-      .then(response => {
-        setMessage(response.message);
+    const currentUser = localStorage.getItem('user')
+    const currentToken = localStorage.getItem('token')
+    if(currentUser && currentToken){
+      const loggedInUser = JSON.parse(currentUser)
+      const loggedInUserToken = JSON.parse(currentToken)
+      setUser(loggedInUser)
+      setToken(loggedInUserToken)
+    }
+  }, []);
+
+  const fetchProducts = () => {
```

Comment:
Great!  Good job creating a function we can use in other components, passed down as a single prop.